release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3009,7 +3009,42 @@ link:https://access.redhat.com/solutions/6114681[{product-title} 4.6.34 containe
+[id="ocp-4-6-34-upgrading"]
 ==== Upgrading
 
-To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
\ No newline at end of file
+To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-35"]
+=== RHBA-2021:2410 - {product-title} 4.6.35 bug fix update
+
+Issued: 2021-06-22
+
+{product-title} release 4.6.35 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2410[RHBA-2021:2410] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:2407[RHBA-2021:2407] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6128971[{product-title} 4.6.35 container image list]
+
+[id="ocp-4-6-35-features"]
+==== Features
+
+[id="ocp-4-6-35-cluster-operator-gatherer-enhancement"]
+===== Insights Operator enhancement
+
+With this update, ClusterOperator `Gather` can now collect logs from pods that are related to unhealthy operators, and/or pods that are living in the same namespace as the operator. This allows the Insights Operator to determine the namespace associated with the operator and include pod logs in the bundle. As a result, the Insights Operator is now collecting logs from unhealthy pods not only related to the operator, but from every pod that lives in the operator namespace.  
+
+[id="ocp-4-6-35-bug-fixes"]
+==== Bug fixes
+
+* Previously, when bringing up a single-stack IPv6 cluster on nodes with IPv4 address, kubelet might have used the IPv4 IP instead of the IPv6 IP for the node IP. Consequently, host network pods would have IPv4 IPs rather than IPv6 IPs, which made them unreachable from IPv6-only pods. This update fixes the node-IP-picking code, which results in the kubelet using the IPv6 IPs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1942488[*BZ#1942488*])
+
+* The fix for link:https://bugzilla.redhat.com/show_bug.cgi?id=1953097[*BZ#1953097*] enabled the CoreDNS `bufsize` plugin with a size of 1232 bytes. Some primitive DNS resolvers are not capable of receiving DNS response messages over UDP that are greater than 512 bytes. Consequently, some DNS resolvers, such as Go's internal DNS library, are unable to receive long-winded DNS responses from openshift-dns. This update sets the CoreDNS `bufsize` to 512 bytes for all servers. As a result, UDP DNS messages are now properly received. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970140[*BZ#1970140*]) 
```

Comment:
s/plugin/plug-in/

Reply:
Hehe "long-winded". Maybe "verbose"?

Reply:
For "openshift-dns", some alternative like "OpenShift DNS" or "the DNS Operator" (whatever is most appropriate for 4.7) would be better.